socorro/unittest/external/es/base.py

```diff
@@ -774,6 +794,15 @@ def test_with_version(self):
     return decorated
 
 
+class SuperSearchWithFields(SuperSearch):
```

Comment:
I actually think it's simpler to have this class in the ``base.py`` file. To me it's the same it terms of instant readability (I'm not used to the ``functools`` module, so I would have to go read the doc of ``partial`` anyway). And with my class, each file using ``SuperSearch`` only has one import to do instead of 3 (functools, SuperSearch and SuperSearchFields).